src/components/CriteriaProvider.tsx

```diff
@@ -40,16 +41,20 @@ const DEFAULT_ELM_STATEMENTS = [
   'Errors'
 ];
 
-function builderModelToCriteria(criteria: BuilderModel): Criteria {
+function builderModelToCriteria(criteria: BuilderModel, label: string): Criteria {
   return {
     id: shortid.generate(),
-    label: `${criteria.type}${Date.now()}`,
+    label,
     modified: Date.now(),
     builder: criteria,
-    statement: criteria.cql
+    statement: label
```

Comment:
Just curious, but is there a reason `label` and `statement` are the same now and do we ever expect them to be different?

Reply:
`label` is how it's displayed in the UI, and `statement` is the way it's referenced in CQL. So when we create a Criteria it's the same, but I think they will be different for imported criteria. (`label` in that case contains the name of the library that it was imported from) 
IIRC `label` is display-only, so changing the above is mostly a display fix for how the UI displays built criteria, which as of today in master looks a little weird: 
<img width="385" alt="image" src="https://user-images.githubusercontent.com/13512036/93387619-86ff7600-f837-11ea-9992-4343f47dac78.png">